views/layout.pug

```diff
@@ -20,6 +20,9 @@ html
                             li.nav-item(class={active: item.active})
                                 a.nav-link(href=item.href)= item.title
                     form.form-inline.my-md-0
+                        if user
+                            div.nav-item
+                                a.nav-link.fa.fa-user.fa-fw(href='/profile/' + user.id)
```

Comment:
Since your adding this button, could you also change the log out button to be an icon?

I played around with it, and this looks nice. You want to make sure to put the font awesome classes on an `i` tag always, for accessibility purposes
```pug
a.btn.btn-outline-secondary.ml-2.px-2(href='/profile/' + user.id, aria-label="Profile")
    i.fa.fa-user.fa-fw(aria-hidden=true)
```

Reply:
Ooh, that looks good